vMenu/Noclip.cs

```diff
@@ -20,6 +20,7 @@ public class NoClip : BaseScript
         private static int Scale { get; set; } = -1;
         private static bool FollowCamMode { get; set; } = false;
 
+        private static LanguageManager LM = new LanguageManager();
```

Comment:
This seems a bit useless? 🤔

Reply:
I tried to add a translation for the NoClip button, but it crashed, seemingly because it was loaded earlier than the main menu.